recipe/meta.yaml

```diff
@@ -250,6 +250,7 @@ outputs:
         - if [[ $($PYTHON -c 'import cv2; print(cv2.__version__)') != $PKG_VERSION ]]; then exit 1; fi  # [unix]
         - python -c "import cv2; assert 'Unknown' not in cv2.videoio_registry.getBackendName(cv2.CAP_V4L)"  # [linux]
         - python -c "import cv2, re; assert re.search('Lapack:\s+YES', cv2.getBuildInformation())"
+  - name: opencv-python
```

Comment:
This won't be enough. If the problem this is trying to solve is the lack of a `opencv-python` package for `pip` to track you need to actually install an empty package with `pip` to set up the metadata.

Reply:
Hi @chrisburr, 

I am unsure what you mean. Should I copy the `py-opencv` section and rename it to `opencv-python`?